clusterman/draining/queue.py

```diff
@@ -395,7 +395,11 @@ def terminate_host(host: Host) -> None:
     logger.info(f'Terminating: {host.instance_id}')
     resource_group_class = RESOURCE_GROUPS[host.sender]
     resource_group = resource_group_class(host.group_id)
-    resource_group.terminate_instances_by_id([host.instance_id])
+
+    try:
+        resource_group.terminate_instances_by_id([host.instance_id])
+    except Exception:
+        logger.error(f'Failed to terminate host: {host.hostname}, possibly already terminated.')
```

Comment:
nit: is this just for human user interactions? in case of use by non-humans, where do these logs go? are they monitored?

Reply:
As far as I can tell these logs aren't processed automatically, and they stay on local host. The script is executed by machines but the logs exist for investigation and debugging if unexpected behaviour occurs. @kawaiwanyelp should have more context on this